multidict/_multidict.c

```diff
@@ -1475,6 +1498,7 @@ module_clear(PyObject *mod)
     return 0;
 }
 
+
```

Comment:
Looks like other sections follow a single-line separator convention between function definitions, only using double-lines where those sections end.
```suggestion
```
Regardless, we shouldn't include unrelated formatting because it surfaces arbitrary chunks of code in the PR diff.

FYI, #1123 proposes plugging a code formatter for C. If you'd like to work on it, this would solve the problem of accidentally changing things in PRs. And it'd be useful to all of us in general.